### ISSUE TITLE
Issues regarding training and generating examples

### ISSUE THREAD

**Issue by xinyuwang1126** [cookielee77/Domain-Adaptive-Text-Style-Transfer#4]
Hi Lee,

I am trying to use the system to adapt sentences to a target domain. I have several questions
1. What is --training_portion? The default value is 1.0, does it mean how many percent of training data the model is going to use?
2. What is --max_epochs? Is it the number of epochs the model is going to train after pre-training?
3. How to generate sentences using a trained model? I suspect I should run train_domain_adapt.py first if I want to do domain adaptation. How should I get adapted sentences on the test set after the training? In evaluation.py it seems like it only calculates the accuracy.

Thank you!

**Comment by cookielee77** [comment]
1. It means how many percentages of training data in the target domain. 

2. No. It means the total epochs for the training, including the pre-training. After "--pretrain_epochs", the model will start to train both style loss and reconstruction loss. 

3. For generating the samples, please refer to: https://github.com/cookielee77/DAST/blob/master/utils.py#L60.
